app/Community/Concerns/ActsAsCommunityMember.php

```diff
@@ -33,10 +33,19 @@ public function activities(): HasMany
     /**
      * @return HasMany<UserGameListEntry>
      */
-    public function gameList(string $type): HasMany
+    public function gameListEntries(): HasMany
     {
-        return $this->hasMany(UserGameListEntry::class, 'user_id', 'ID')
-            ->where('SetRequest.type', $type);
+        return $this->hasMany(UserGameListEntry::class, 'user_id', 'ID');
+    }
+
+    /**
+     * @return HasMany<UserGameListEntry>
+     */
+    public function gameLists(string $type): HasMany
```

Comment:
This feels wrong. The name implies it would return multiple lists instead of multiple entries. Recommend overloading the first function:
```
    public function gameListEntries(?string $type = null): HasMany
    {
        $query = $this->hasMany(UserGameListEntry::class, 'user_id', 'ID');
        
        if ($type !== null)
        {
            $query->where('SetRequest.type', $type);
        }
        
        return $query;
    }
```

Reply:
Good idea. Updated in latest.